### ISSUE TITLE
Can the extension have intelligent autohide feature as in dash to dock

### ISSUE THREAD

**Issue by mrb2604** [sunwxg/gnome-shell-extension-floatingDock#16]
Can the extension have intelligent autohide feature with some customisation as in dash to dock extension.

**Comment by sunwxg** [comment]
Could you give a detail description for the autohide feature? I don't catch the idea, when to hide, when to show, and what to be hided?

**Comment by SirBubbles** [comment]
I'd add a plus 1 to this idea. It basically means that when a window has focus like Firefox, then hide the dock. When you mouse over where the dock should be, ie, the bottom of the screen, then the dock shows.
IE, I installed your extension because DashToDock currently doesn't support gnome40. Yours does. However, autohide is one obvious missing feature that your extension doesn't yet have. If it did, it would be awesome.

**Comment by Nebrit** [comment]
I totally agree, it would be necessary that when there is an active window it disappears completely and when the mouse is positioned in the area where the "dot" is, it appears; (autohide)
